test/dotnet.Tests/CommandTests/ToolInstallGlobalOrToolPathCommandTests.cs

```diff
@@ -975,6 +989,16 @@ public void SetPermission(string path, string chmodArgument)
             {
             }
         }
+
+        private string _nugetConfigWithInvalidSources = @"{
+<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+  <packageSources>
+    <add key=""nuget"" value=""https://api.nuget.org/v3/index.json"" />
+    <add key=""invalid_source"" value=""https://invalidsource.com/v3/index.json"" />
```

Comment:
@dsplaisted Is there a standard feed we can use for testing an invalid source? I couldnt find one. Im not sure I like relying on this website but I dont have anything better unless we have something internal we can use for this.